relay-server/src/actors/project_upstream.rs

```diff
@@ -358,7 +361,11 @@ impl UpstreamProjectSourceService {
                             .remove(&key)
                             .unwrap_or(ErrorBoundary::Ok(None))
                             .unwrap_or_else(|error| {
-                                relay_log::error!(error, "error fetching project state {key}");
+                                relay_log::error!(
+                                    tags.project_key = key.as_str(),
+                                    error,
+                                    "error fetching project state: deserialization error"
```

Comment:
I took the deserialization part from the docs of the `ProjectState::err` error created below:
https://github.com/getsentry/relay/blob/b33f76e4c9926b90cc9b56e0a6e645601e8c690d/relay-server/src/actors/project.rs#L139-L140